packages/overlays/test/LocalOverlayController.test.js

```diff
@@ -515,6 +515,51 @@ describe('LocalOverlayController', () => {
   });
 
   describe('a11y', () => {
+    it('adds no relationship to the invoker', async () => {
+      const controller = new LocalOverlayController({
+        contentTemplate: () => html`
+          <p style="width: 80px; height: 20px;"></p>
+        `,
+        invokerTemplate: () => html`
+          <button style="width: 100px; height: 20px;" @click=${() => controller.show()}>
+            Invoker
+          </button>
+        `,
+      });
+      expect(controller.invokerNode.hasAttribute('aria-describedby')).to.be.false;
```

Comment:
```suggestion
      expect(controller.invokerNode).to.not.have.attribute('aria-describedby');
````
will give a much nicer error message and "reads" more like a normal sentence